tests/test_loadDiaCatalogs.py

```diff
@@ -59,13 +59,8 @@ def setUp(self):
         self.db_file_fd, self.db_file = tempfile.mkstemp(
             dir=os.path.dirname(__file__))
 
-        self.apdbConfig = ApdbSqlConfig()
-        self.apdbConfig.db_url = "sqlite:///" + self.db_file
-        self.apdbConfig.dia_object_index = "baseline"
-        self.apdbConfig.dia_object_columns = []
-
-        Apdb.makeSchema(self.apdbConfig)
-        self.apdb = ApdbSql(config=self.apdbConfig)
+        self.apdbConfig = ApdbSql.init_database(db_url="sqlite:///" + self.db_file)
```

Comment:
Is `init_database` preferred to `create_sql`?

Reply:
`create_sql` and all stuff in `lsst.dax.apdb.scripts` is for CLI use (`apbd-cli` or any other CLI might need that). For regular Python code `init_database` is the main interface.